.github/workflows/open-pull-request-for-icon-update.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04 # v4.2.3
         with:
           branch: 'hds-icon-export'
+          branch-suffix: 'random'
```

Comment:
I remember using `short-commit-hash` in the past but not opinionated

Reply:
i would be fine with that too. it's almost a curse of too many options that all seem to do the same thing.